src/tslint-cli.ts

```diff
@@ -18,161 +18,163 @@
-    outputStream = process.stdout;
-}
-
-if (argv.v !== undefined) {
-    outputStream.write(Lint.Linter.VERSION + "\n");
-    process.exit(0);
-}
-
-if ("help" in argv) {
-    outputStream.write(optimist.help());
-    const outputString = `
-tslint accepts the following commandline options:
-
-    -c, --config:
-        The location of the configuration file that tslint will use to
-        determine which rules are activated and what options to provide
-        to the rules. If no option is specified, the config file named
-        tslint.json is used, so long as it exists in the path.
-        The format of the file is { rules: { /* rules list */ } },
-        where /* rules list */ is a key: value comma-seperated list of
-        rulename: rule-options pairs. Rule-options can be either a
-        boolean true/false value denoting whether the rule is used or not,
-        or a list [boolean, ...] where the boolean provides the same role
-        as in the non-list case, and the rest of the list are options passed
-        to the rule that will determine what it checks for (such as number
-        of characters for the max-line-length rule, or what functions to ban
-        for the ban rule).
-
-    -o, --out:
-        A filename to output the results to. By default, tslint outputs to
-        stdout, which is usually the console where you're running it from.
-
-    -r, --rules-dir:
-        An additional rules directory, for additional user-created rules.
-        tslint will always check its default rules directory, in
-        node_modules/tslint/build/rules, before checking the user-provided
-        rules directory, so rules in the user-provided rules directory
-        with the same name as the base rules will not be loaded.
-
-    -s, --formatters-dir:
-        An additional formatters directory, for user-created formatters.
-        Formatters are files that will format the tslint output, before
-        writing it to stdout or the file passed in --out. The default
-        directory, node_modules/tslint/build/formatters, will always be
-        checked first, so user-created formatters with the same names
-        as the base formatters will not be loaded.
-
-    -t, --format:
-        The formatter to use to format the results of the linter before
-        outputting it to stdout or the file passed in --out. The core
-        formatters are prose (human readable), json (machine readable)
-        and verbose. prose is the default if this option is not used. Additonal
-        formatters can be added and used if the --formatters-dir option is set.
-
-    -v, --version:
-        The current version of tslint.
-
-    -h, --help:
-       Prints this help message.`;
-    outputStream.write(outputString);
-    process.exit(0);
-}
-
-// when provided, it should point to an existing location
-if (argv.c && !fs.existsSync(argv.c)) {
-    console.error("Invalid option for configuration: " + argv.c);
-    process.exit(1);
-}
+module Lint {
```

Comment:
this is a pretty bad diff. I just moved everything into an internal module to avoid #570.